src/substrate/substrate.py

```diff
@@ -145,13 +163,21 @@ def main():
 		help='[start, stop]',
 		metavar='ACTION'
 	)
-	parser.add_argument('-c', '--config', dest='path')
+	parser.add_argument('-i', '--input' , dest='input_config', default='{}', type=str, nargs='?', const=1)
+	parser.add_argument('-p', '--path'  , dest='path')
```

Comment:
I prefer something like this:

`parser.add_argument('-c', '--config' , dest='input_config', default='{}', type=str, nargs='?', const=1)`
`parser.add_argument('-f', '--config-file' , dest='path')`